### PR TITLE
Add service notes on Freebox integration

### PR DESCRIPTION
**Pull request by SNoof85**
**Description:**
Add few lines about the new service to reboot the freebox.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#29525

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/freebox.markdown [hunk 1]
```diff
@@ -18,6 +18,7 @@ There is currently support for the following device types within Home Assistant:
 * [Sensor](#sensor) with traffic metrics
 * [Device tracker](#presence-detection) for connected devices
 * [Switch](#switch) to control WiFi
+* [Service](#service) to reboot the Freebox router
```
Comment (frenck) [inline source/_integrations/freebox.markdown]: Service is not an entity type, please remove it from this list. (This is not a TOC).

File: source/_integrations/freebox.markdown [hunk 2]
```diff
@@ -94,6 +95,10 @@ refreshes the devices states.
 This platform offers you sensors to monitor a Freebox router. The monitored conditions are
 instant upload and download rates in KB/s.
 
+## Service
+
+A service is available to reboot your Freebox router.
```
Comment (frenck) [inline source/_integrations/freebox.markdown]: Describe the service call. For a reference, check some other page on how they have added it.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR, @SNoof85!

I've left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/freebox.markdown (hunk 1 above)
Comment (SNoof85) [reply source/_integrations/freebox.markdown]: removed 👍

File: source/_integrations/freebox.markdown (hunk 2 above)
Comment (SNoof85) [reply source/_integrations/freebox.markdown]: Is that better ?

**Review by frenck — APPROVED** [review]

File: source/_integrations/freebox.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/freebox.markdown]: Yep, thanks!

**Pull request merged**